src/ui/interface.js

```diff
@@ -1629,11 +1629,22 @@ var UI = class UI {
 	    this.xrayQueue(creaID);
 		if (creaID > 0) 
 			var $queueItem = this.$queue.find('.vignette[creatureid="' + creaID + '"]:first');
-			if($queueItem.length > 0 && $queueItem.data("playingEffect") != true) {
-			    $queueItem.data("playingEffect",true);
-                $queueItem.effect( "bounce", { distance:-20,times: 1 },400,
-                    setTimeout(function(){$queueItem.data("playingEffect",false); }, 600)
-                );
+			if ($queueItem.length > 0) {
+			    if ($queueItem.data("playingEffect") != true) {
+                    $queueItem.data("playingEffect",true);
```

Comment:
Coding style consistency: you need to use more spaces, especially after commas that are followed by something.